openedx/core/djangoapps/oauth_dispatch/jwt.py

```diff
@@ -61,6 +61,9 @@ def create_jwt_from_token(token_dict, oauth_adapter, use_asymmetric_key=None):
     access_token = oauth_adapter.get_access_token(token_dict['access_token'])
     client = oauth_adapter.get_client_for_token(access_token)
 
+    # use a JWT specific expiration
+    token_dict['expires_in'] = settings.JWT_ACCESS_TOKEN_EXPIRE_SECONDS
```

Comment:
This is causing a unit test failure in one of the openedx test shards, which uses `--ds=cms.envs.test`.

Reply:
Thanks. I think I'm going to inline the annotations and default like you did here: https://github.com/openedx/edx-django-utils/blob/4559a523f24f2d3c922b40536f7706b4e66c404e/edx_django_utils/monitoring/internal/middleware.py#L426-L432

Reply:
I moved the annotations and default.